lang/cs/Org.Apache.REEF.Common/Files/REEFFileNames.cs

```diff
@@ -338,12 +339,21 @@ public string GetPidFileName()
         }
 
         /// <summary>
+        /// The filename of the file that contains the driver name server address and port.
+        /// </summary>
+        /// <returns>File name that contains the dfs path for the DriverNameServerEndpoint</returns>
+        public string GetDriverJavaBridgeEndpoint()
+        {
+            return DRIVER_JAVA_BRIDGE_ENDPOINT_FILE_NAME;
+        }
+
+        /// <summary>
         /// </summary>
         /// <returns>File name that contains the dfs path for the DriverHttpEndpoint</returns>
         [Unstable("0.13", "Working in progress for what to return after submit")]
-        public string DriverHttpEndpoint 
-        { 
-            get { return DRIVER_HTTP_ENDPOINT_FILE_NAME; } 
+        public string DriverHttpEndpoint
```

Comment:
rename to DriverHttpEndpointFileName?

Reply:
That particular code does not belong to this PR; renaming the field will require changes in other files not related to this update. Let's do it later in a separate PR.